editor-components/header/header22/header22.tsx

```diff
@@ -2,95 +2,390 @@ import * as React from "react";
+            <div className={this.decorateCSS("max-content")}>
+              <div className={this.decorateCSS("slider-parent")} style={{ minHeight: minHeight + "px" }} >
+                <ComposerSlider
+                  {...settings}
+                  className={this.decorateCSS("carousel")}
+                  ref={this.getComponentState("slider-ref")}>
+                  {slider.map((item: SliderObject, index: number) => {
+                    const isActive = this.getComponentState("activeSlide") === index;
+                    const leftImageExist = item.left_image;
+                    const rightImageExist = item.right_image;
+                    const middleClass = leftImageExist ? "middle-content" : "middle-content2";
+                    const middleClassWithPadding = (!leftImageExist && !rightImageExist) || (!leftImageExist && !this.getPropValue("dots")) ? "middle-content3" : middleClass;
+                    return (
+                      <div
+                        className={this.decorateCSS("sliders")}
+                        key={index}>
+                        <div className={this.decorateCSS("slider")}>
+                          {leftImageExist && (
+                            <div className={this.decorateCSS("left-content")}>
+                              <img
+                                className={`${this.decorateCSS("left-image")} ${(animation && isActive) ? this.decorateCSS("left-animation") : ""}  `}
+                                src={item.left_image}
+                                alt=""
+                              />
+                            </div>
+                          )}
+
+                          <div
+                            className={`${this.decorateCSS(middleClassWithPadding)}
+                          } ${animation && isActive ? this.decorateCSS("mid-right-animation") : ""}  `}>
+                            <Base.VerticalContent className={this.decorateCSS("text-wrapper")}>
+                              {hasDivider && <div className={this.decorateCSS("divider")} />}
+                              {this.castToString(item.title) && <Base.SectionTitle className={this.decorateCSS("title")}>{item.title}</Base.SectionTitle>}
+
+                              {item.button.map((buttonItem: any, indexButton: number) => {
+                                const buttonText = this.castToString(buttonItem.buttonText);
+                                if (buttonText) {
+                                  return (
+                                    <div className={this.decorateCSS("link-button-container")}>
+                                      <ComposerLink
+                                        className={this.decorateCSS("link-button")}
+                                        key={`hdr-22-${indexButton}`}
+                                        path={buttonItem.link}>
+                                        <button className={this.decorateCSS("button")}>{buttonText}</button>
```

Comment:
use base button

Reply:
we are not using base botton for this button